tests/systemd/prepare.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+qm_service_file=$(systemctl show -P  FragmentPath qm)
```

Comment:
@Yarboa please move all the nested QM unit files attributes to plan prepare
Use only the relevant attributes important to to the test.
Do not forget to return unit file to its previous condition

Reply:
Talking to yourself?

Reply:
Yep, sorry :)

Reply:
Done